instana/instrumentation/flask/vanilla.py

```diff
@@ -17,6 +17,21 @@
 path_tpl_re = re.compile('<.*>')
 
 
+def extract_custom_headers(span, headers, format):
```

Comment:
If the extract function is the same for both vanilla and with_blinker, then we could put this function into a separate file and include it here and into with_blinker.

Reply:
@Ferenc- Is it okay if I add the function in [this](https://github.com/instana/python-sensor/blob/master/instana/instrumentation/flask/common.py) file?

Reply:
I think it could fit there, yes.